typeshed_client/parser.py

```diff
@@ -430,17 +424,40 @@ def generic_visit(self, node: ast.AST) -> Iterable[NameInfo]:
         raise InvalidStub(f"Cannot handle node {ast.dump(node)}", self.file_path)
 
 
-class LiteralEvalVisitor(ast.NodeVisitor):
-    """Visitor to evaluate the truthiness of a ``test`` expression in an ``ast.Compare`` node.
+def evaluate_expression_truthiness(
+    expr: ast.expr, *, ctx: SearchContext, file_path: Optional[Path], is_py_file: bool
```

Comment:
I think we can do:
```suggestion
    expr: ast.expr, *, ctx: SearchContext, file_path: Optional[Path], is_py_file: bool = False
```
Since external users (such as you) probably are using this function on actual stub files. It's also the more conservative option since `is_py_file=True` would suppress more errors.

Reply:
Looking at it a bit more deeply, isn't the argument duplicative? We could probably infer this information from the extension of `file_path` inside the function?